ruby_on_rails/app_initialisation.md

```diff
@@ -117,6 +117,7 @@ create the initializer for figaro in `config/initializers/figaro.rb`:
 
   ```ruby
   config.action_view.raise_on_missing_translations = true # uncomment
+  config.active_record.verbose_query_logs = true # add
```

Comment:
so if i understand correctly, that would help in tests, right?

Reply:
When debugging tests (or using tests to debug) and a `Rails.logger` is defined (which it isn't by default in the test environment) it will help.